src/entry-points/local-file-player/App.svelte

```diff
@@ -45,14 +44,14 @@ along with Jump Cutter Browser Extension.  If not, see <https://www.gnu.org/lice
     }
 
     for (const file of inputEl.files) {
-      files.push(file);
+      files = [...files, file];
     }
 
     playFile(0);
```

Comment:
Probably need to change this as well? Otherwise it always resets `currFileInd`.

Reply:
I can do it myself later.

Reply:
Yes, line50 needs to change as well. this code will play the file once we add a file into playlist. But it will also replay the video when we add other file into the playlist.